### ISSUE TITLE
Build landing page

### ISSUE THREAD

**Issue by bhekanik** [bhekanik/romulus#4]
You can use components from the following UI libraries to build a great looking site very fast: 
- https://ui.aceternity.com/components
- https://magicui.design/docs/components/marquee
- https://syntaxui.com/
- https://motion-primitives.com/
- https://animata.design/
- https://ui.lukacho.com/
- https://www.hyperui.dev/

**Comment by AdrianTNare** [comment]
do we have any design guidelines on what this landing page is supposed to look like ?

**Comment by bhekaniatdba** [comment]
Nope, at the moment we just want a skeleton. So use chatgpt to get some copy, look around other saas landing pages and then build something that makes sense. We can refine it later.
